src/charts/bar.tsx

```diff
@@ -20,7 +20,9 @@ interface BarOptions {
 export const semioticBarChart = (
   data: Dx.Datapoint[],
   schema: Dx.Schema,
-  options: BarOptions
+  options: BarOptions,
+  colorHashOverride?: Object,
```

Comment:
Is there a more specific interface we can use for this instead of `Object`?

Reply:
Updated to `{ key?: string }`